src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -251,14 +250,14 @@ public LightningStrike strikeLightningEffect(Location loc, boolean isSilent) {
      * Per-chunk spawn limits on various types of entities.
      */
     private int monsterLimit, animalLimit, waterAnimalLimit, ambientLimit;
-    private Map<Integer, GlowStructure> structures;
+    private Map<Long, GlowStructure> structures;
```

Comment:
Perhaps TLongObjectMap could be better here?

Reply:
You're talking about the trove stuff, correct? I hadn't seen it used anywhere else in the code base, and this is already pretty lean as it is. I try not to do too much premature optimization, and in this case I don't think we'll be storing millions of items in the map. In addition, while Trove is stable, it seems within the last couple years [other libraries](http://java-performance.info/hashmap-overview-jdk-fastutil-goldman-sachs-hppc-koloboke-trove-january-2015/) outperform it.